x-pack/plugins/actions/server/builtin_action_types/email.ts

```diff
@@ -71,25 +74,36 @@ function validateConfig(
 ): string | void {
   const config = configObject;
 
-  // Make sure service is set, or if not, both host/port must be set.
-  // If service is set, host/port are ignored, when the email is sent.
+  // If service is set as JSON_TRANSPORT_SERVICE or EXCHANGE, host/port are ignored, when the email is sent.
   // Note, not currently making these message translated, as will be
   // emitted alongside messages from @kbn/config-schema, which does not
   // translate messages.
   if (config.service === JSON_TRANSPORT_SERVICE) {
     return;
+  } else if (config.service === AdditionalEmailServices.EXCHANGE) {
```

Comment:
nit: maybe we can remove `AdditionalEmailServices.EXCHANGE` from `CUSTOM_CONFIG_SERVICES` and then rename `CUSTOM_CONFIG_SERVICES` to something more specific like `CUSTOM_HOST_PORT_SERVICES`?